src/main/java/com/okta/tools/OktaAwsCliAssumeRole.java

```diff
@@ -312,8 +312,8 @@ private AssumeRoleWithSAMLRequest chooseAwsRoleToAssume(String samlResponse) thr
                 i++;
             }
         }
-        if (awsRoleToAssume != null && j == -1) {
-            System.out.println("No match for role " + awsRoleToAssume);
+        if ((awsRoleToAssume != null && !awsRoleToAssume.isEmpty()) && j == -1) {
+            System.out.println("\ncould not find the role " + awsRoleToAssume + "\n specified in the config.properties file.");
```

Comment:
nit: We should capitalize this error message + keep the error string as one line. So we can remove the `\n` newline after `awsRoleToAssume`

Reply:
**awsRoleToAssume** isn't necessarily from the config.properties file

Reply:
I disagree with making this logging change.